helpers/analyticsHelper.js

```diff
@@ -378,6 +378,16 @@ export const EVENTS = {
     },
     ActionClicked: 'Outreach - Action Clicked',
   },
+  CandidateWebsite: {
+    Started: 'Candidate Website - Started',
+    Continued: 'Candidate Website - Continued',
+    Published: 'Candidate Website - Published',
+    Unpublished: 'Candidate Website - Unpublished',
+    Edited: 'Candidate Website - Edited',
+    StartedDomainSelection: 'Candidate Website - Started domain selection',
+    SelectedDomain: 'Candidate Website - Selected domain',
+    PurchasedDomain: 'Candidate Website - Purchased domain',
```

Comment:
We should probably track all the steps in the creation flow.
They don't change the URL, but the user progresses through the steps

https://github.com/thegoodparty/gp-webapp/blob/develop/app/(candidate)/dashboard/website/create/components/WebsiteCreateFlow.js#L242C13-L283C15